src/levels/remote/pushArgs2.js

```diff
@@ -55,7 +55,7 @@ exports.level = {
               "",
               "`git push origin <source>:<destination>`",
               "",
-              "This is commonly referred to as a colon refspec. Refspec is just a fancy name for a location that git can figure out (like the branch `foo` or even just `HEAD~1`)",
+              "This is commonly referred to as a colon refspec. Refspec is just a fancy name for a location that git can figure out (like the branch `foo` or even just `HEAD~1`)".,
```

Comment:
oh just FYI, the period at the end of the line here was outside the `"` so it caused a json parsing error and some issues when landing. I had to fix but nbd

Reply:
@pcottle Oh whoops, I don't know how I missed that :smile: thanks for fixing!